runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/Work.java

```diff
@@ -147,16 +150,25 @@ private static LatencyAttribution.Builder createLatencyAttributionWithActiveLate
     return latencyAttribution;
   }
 
+  public RefreshableWork refreshableView() {
```

Comment:
can we remove this? can it just auto-cast? or in the case where perhaps it isn't for streams just do it via a cast there?

Reply:
done